docs/fedora-livemedia.ks

```diff
@@ -389,4 +388,13 @@ syslinux
 -gfs2-utils
 -reiserfs-utils
 
+# This package is needed to boot the iso on UEFI
+shim
+shim-ia32
+grub2
+grub2-efi
+grub2-efi-*-cdboot
+grub2-efi-ia32
```

Comment:
Shouldn't this be `grub2-efi-*`?

Reply:
No, it's just the basics needed to get it to boot on UEFI.

Reply:
Just for clarity again, grub2-efi is a magic package that selects grub2-efi-x64 on f29.